modules/developer_manual/pages/general/backporting.adoc

```diff
@@ -118,51 +80,44 @@ Pushing: ...
 ...
 ----
 
-NOTE: Please keep in mind that this is an example and you have to adapt the commit hash and the
-target branch accordingly.
+NOTE: Please keep in mind that this is an example and you have to adapt the commit hash and the target branch accordingly.
+
+The script lists quantity and commits to be backported and the current cherry-pick in process. This can be helpful in case there is a conflict and you manually continue after the conflict has been resolved.
 
-The script lists quantity and commits to be backported and the current commit in process.
-This can be helpful in case there is a conflict and you manually continue after the
-conflict has been resolved.
+When the script completes, go to GitHub, where it will suggest that you make a PR from pushed branch.
```

Comment:
your script looks like it will now automatically open the browsxer to the link for creating the PR.

Reply:
you are right, fixing asap...

Reply:
fixed